public/app/features/commandPalette/actions/dashboard.nav.actions.ts

```diff
@@ -1,21 +1,24 @@
 import { Action } from 'kbar';
 
 import { locationUtil } from '@grafana/data';
-import { locationService, getBackendSrv } from '@grafana/runtime';
+import { locationService } from '@grafana/runtime';
+import { getGrafanaSearcher } from 'app/features/search/service';
 
 async function getDashboardNav(parentId: string): Promise<Action[]> {
-  const data: Array<{ type: string; title: string; url: string }> = await getBackendSrv().get('/api/search');
+  const data = await getGrafanaSearcher().search({
+    kind: ['dashboard'],
+    query: '*',
+    limit: 500,
+  });
 
-  const goToDashboardActions: Action[] = data
-    .filter((item) => item.type === 'dash-db')
-    .map((item) => ({
-      parent: parentId,
-      id: `go/dashboard/${item.url}`,
-      name: `Go to dashboard ${item.title}`,
-      perform: () => {
-        locationService.push(locationUtil.stripBaseFromUrl(item.url));
-      },
-    }));
+  const goToDashboardActions: Action[] = data.view.map((item) => ({
+    parent: parentId,
+    id: `go/dashboard/${item.url}`,
```

Comment:
does this just need to be unique?  if yes, let's use
```suggestion
    id: `go/dashboard/${item.uid}`,
```

Reply:
Hmm, we use this partially for the usage tracking as well. Having the URL was something a bit more readable. I'll just track the ID and name if we ever want to dive in a little - either that or we find out it's worthless to track something specific to instances and we just group the go to dashboard actions as one thing.

Reply:
makes sense